portfolio/src/main/webapp/experience.html

```diff
@@ -0,0 +1,100 @@
+		operations of a non-profit organization, analyzed the organization’s 
+		logic model, reviewed the program handbooks, and provided new suggestions.
+		At the end of the program, I hosted a 45-minutes group session with the 
+		7 high school participants 
+		and created an interactive, wireless device, using Arduino parts, 
+		that lights up when a phone receives notifications.
+		</p>
+		<h4>ZBS POS Retail and Restaurant Solution</h4>
+		<p><i>June 2017</i></p>
+		<p>
+		I was a technical support team representative that 
+		implemented the restaurant’s ordering and menu technology, 
+		provided strategic guidance to restaurant managers on creating 
+		an optimal layout for the menu system, and
+		connected restaurant’s printing devices to 
+		all computers through long-distance control via TeamViewer.
+		</p>
+		<hr>
+		<h2>Non-technical Experiences</h2>
+
+		<h4>Columbia University Dodge Fitness Center</h4>
+		<p><i>January 2019 - Present</i></p>
+		<p>
+		I am an intrmural sports official that regulate, officiate, 
+		score keep statistics for basketball games, assist supervisors with 
+		court setups, and help resolve on-court conflicts.
+		</p>
+		<h4>Columbia University Butler Library</h4>
+		<p><i>October 2019 - March 2020</i></p>
+		<p>
+		I was a student assistant in the Periodicals & Microforms Reading Room (PRRM)
+		that helped patrons with loading and seting up microfilm readers, 
+		media players and scanners, reshelved and cataloged periodicals.
+		</p>	
+		<h4>Lucky Dragon Restaurant</h4>
+		<p><i>June 2016 - Present</i></p>
+		<p>
+		I started off as a waitress and a front-desk receptionist that 
+		answered and packed phone call orders, received payments at checkout, 
+		welcomed and served dine-in customers, 
+		communicated with customers about the quality of food, 
+		and translated and reported the feedback to the manager.	
+		In 2018, I started to pick up some more technical responsibilities such as 
+		customize and maintain the ordering system, install the computer system, 
+		revise and design take-out menu using Photoshop, 
+		and provide ongoing technical support virtually via TeamViewer during the school year.
+
+		</p>
+		<h4>Global Youth Mentorship Initiative (GYMI)</h4>
+		<p><i>October 2018 - Present</i></p>
+		<p>
+		I instructed yoga to 35 students and mentored 3 
+		underprivileged middle school students 
+		in the 2-week UN summer camp in Guizhou, China. 
+		During the school year, I help designed a social-emotional learning curriculum
+		for middle school and high school students in New York City.
+		</p>
+		<h4>Matriculate</h4>
+		<p><i>March 2019 - May 2020</i></p>
+		<p>
+		As an advising fellow, I advised 2 low-income, high achieving high school 
+		students about college application process for three semesters, 
+		starting with drafting a balanced college list to the final stage of 
+		commiting to a college.
+		</p>
+	</div>
+</body>
+</html>
```

Comment:
Could you separate the portfolio content commits into a separate PR from the servlet changes?

Reply:
Yes. I committed the portfolio stuff into a different branch. I thought those wouldn't show up under this branch, but they did. Is there any way I could remove those commits from this branch and have the portfolio commits be only visible under the branch named "pages?"

Reply:
Did you rebase your new branch from master? After committing here (in github) you need to do something like this:

* `git checkout master` in cloudshell
* `git pull` in cloudshell (to update the `master` branch locally to be up to date with respect to `origin/master` (the branch on GitHub)
* `git checkout <your current banch here>`
* `git rebase master` to make the changes here be w.r.t. master branch

Reply:
I did not rebase from master. So what I did was that I created a new branch for servlet and then went back and committed the portfolio changes under the "pages" branch. Then, I switched back to servlet branch and committed the hello message change. I have not push from the "pages" branch yet. 

Will rebase remove my portfolio commits from the servlet and pages branch, or just the servlet branch?

Reply:
You didn't make the pages branch into a PR. Why? Maybe you should fix that.

I think that you created a new `servlet` branch from the `pages` branch. That explains why changes from `pages` are included in the `servlet` branch.

You're welcome to have multiple in-progress PRs but each one should be branched from `master`.

Also don't forget that the branches on GitHub and in your client are not automatically kept in sync. You need to manually synchronize them with `git pull` or `git push`. I suggest that you draw a graph with nodes corresponding to the different branches that you have and think about how actions move state between nodes.

Reply:
I was still working on the portfolio, so I didn't create a PR for it yet. But I just created a separate PR for the portfolio content as well. 

I think that's exactly what happened. I created a branch from pages, instead of master. In this case, could I just delete the servlet branch and recreate it from master?

Thanks for the reminder.

Reply:
Yes. Make a new branch from master. And then you can use `git cherry-pick` to add the last servlet change from this branch into the new branch, push the new branch, etc. and abandon this one.

Reply:
Thanks for the help. I have deleted this branch and created a new one with the server content.